components/CreateGiftCardsForm.js

```diff
@@ -341,7 +350,7 @@ class CreateGiftCardsForm extends Component {
     return (
       <Box>
         <Flex flexDirection="column" mb="2em">
-          <label style={{ width: '100%' }}>
+          <Strong style={{ width: '100%' }}>
```

Comment:
We should link the input with `htmlFor` rather than removing the label

Reply:
@Betree : Same as my as my [above comment](https://github.com/opencollective/opencollective-frontend/pull/5961#discussion_r592493219), I was thinking not to use `label` here as we are using it as a styling element. What `<input>` element should we link it to? 🤔 Correct me if I am wrong, but I didn't find any except our custom `StyledMultiEmailInput` which is a `Container`.

Reply:
The editor in `StyledMultiEmailInput` has a `role="textbox"`, so it might be safe to pass it an `id` and use that as a value for `htmlFor`.

Reply:
@Betree : For this one, I was looking at the `StyledMultiEmailInput` and that doesn't seem to be a input element although it says `StyledMultiEmailInput` in the name. Even when I checked in the rendered DOM tree, couldn't find an input element. So I was thinking we should keep this one as a `Strong`. 

![image](https://user-images.githubusercontent.com/12435965/111212036-5ac97580-858c-11eb-82c6-b615460f9daf.png)

Reply:
Since it has `role="textbox"`, I think it's safe to treat it as an `input`

Reply:
@Betree : Thanks I agree. Sorry, I think you've mentioned this before; but somehow I missed the point. 😺